code/modules/projectiles/guns/projectile/automatic.dm

```diff
@@ -223,4 +224,20 @@
 	origin_tech = "combat=5;materials=1;syndicate=2"
 	mag_type = "/obj/item/ammo_box/magazine/tommygunm45"
 	fire_sound = 'sound/weapons/Gunshot_smg.ogg'
-	can_suppress = 0
\ No newline at end of file
+	can_suppress = 0
+	burst_size = 4

```

Comment:
fire_delay of 1 here (not that it matters, but since you said you're going for consistency/feature/matching update).